lib/internal/crypto/util.js

```diff
@@ -2,15 +2,11 @@
 
 const Buffer = require('buffer').Buffer;
 
-function toBuf(str, encoding) {
-  if (typeof str === 'string') {
-    if (encoding === 'buffer' || !encoding)
-      encoding = 'utf8';
-    return Buffer.from(str, encoding);
-  }
-  return str;
-}
-exports.toBuf = toBuf;
+exports.toBuf = function toBuf(str, encoding) {
+  return typeof str === 'string' ?
```

Comment:
Again, was much clearer before. The necessity of the change is not evident to me.